eureka2-core/src/main/avro/replication-schema.avdl

```diff
@@ -21,20 +23,31 @@ protocol ReplicationProtocol {
         boolean sendSnapshot;
     }
 
-    record RegisterCopy {
+    @namespace("com.netflix.eureka2.protocol.interest")
+    record AddInstance {
```

Comment:
As we now share many types across different protocols, it makes sense to put them all into common "model" schema and leave protocol schemas with protocol specific declarations only (like ReplicationMessages below). This way we can avoid duplicating definitions.